### PR TITLE
Improve code quality in PdfConverter.java

### PR DESCRIPTION
**Pull request by gwynethguo**
# Changes
- Make methods <= 30 LOC

### CODE REVIEW

File: src/main/java/seedu/address/model/PdfConverter.java [hunk 1]
```diff
@@ -65,19 +65,21 @@ public class PdfConverter {
 
     private PDPageContentStream contentStream;
 
-    /**
-     * Exports {@code key}'s task and score list in the form of a PDF file from this {@code Mathutoring}.
-     */
-    public PDDocument exportProgress(Student key) throws IOException {
-        requireAllNonNull(key);
+    public void setUp() throws IOException {
```
Comment (astraxq) [inline src/main/java/seedu/address/model/PdfConverter.java:68]: setup is a single word, dont have the camelcase it.

**Review by astraxq — COMMENTED** [review]
Overall, good job! Code looks a lot cleaner. Just a minor name change 😄

**Comment by CarrieLi1015** [comment]
LGMT! I have the same suggestion as Brian. :)

File: src/main/java/seedu/address/model/PdfConverter.java (hunk 1 above)
Comment (gwynethguo) [reply src/main/java/seedu/address/model/PdfConverter.java]: noted!

**Pull request merged**